### ISSUE TITLE
Giant grabber crane

### ISSUE THREAD

**Issue by gaymeowing** [typical-developers/oaklands-feedback#970]
### Roblox Username

kalrnlo

### What do you think can have quality-of-life improvements?

unloading

### How do you think things can be improved?

Currently theres no way to automate unloading fully in game, outside of abusing the fact that driving over an alignment conveyor will unload a vehicle pretty much fully in like an instant.

So it'd be cool to see a giant grabber on like a crane that can be used to grab all the items out of like a single trailer or vehicle, and dump them somewhere. The crane could be either on tracks (like cranes at ports), or a standing static crane that can rotate 360 degrees.
Cranes would be operated via a joystick attached to them, or via logic.

Resizing the front and back sides of a crane will increase/decrease the size of the grabber. but if the crane is on a track resizing the left and right sides will extend the cranes track. 
If a standing static crane is added you'd be able to resize it from the top and bottom to increase/decrease its height.

reference image of a crane with a magnet:
![image0-3](https://github.com/user-attachments/assets/b0cb284f-dc0e-4f82-8724-0236c5dc048e)

**Comment by Silbersix** [comment]
Unloading already happens in 2 ways, one with client side lag one without.

Driving through a collision toggled storage drawer: you go from there once the material is in a drawer. 

or the classic: respawn the truck. 

A crane would be cool, but just adding to acknowledge that there are other methods

**Comment by gaymeowing** [comment]
> Unloading already happens in 2 ways, one with client side lag one without.
> 
> Driving through a collision toggled storage drawer: you go from there once the material is in a drawer.
> 
> or the classic: respawn the truck.
> 
> A crane would be cool, but just adding to acknowledge that there are other methods

True but those are more of just glitches that sorta abuse the games mechanics for stuff, not really actual methods provided by the game.

**Comment by Silbersix** [comment]
not necessarily, if these were glitches they would be patched. colliders on storage drawers was an intentionally added mechanic.

**Comment by HooferDevelops** [comment]
> colliders on storage drawers was an intentionally added mechanic

it actually wasn't! I found out about it after release and I decided not to "patch it" because it seemed pretty nifty.
